PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmitModel.java

```diff
@@ -66,6 +67,7 @@
     private Map<String, Artifact> artifactHashMap = new HashMap<>();
 
     private SparkSubmissionParameter submissionParameter;
+    private SparkSubmitAdvancedConfigModel advancedConfigModel;
```

Comment:
Same with above

---
In reply to: [117161326](https://github.com/Microsoft/azure-tools-for-java/pull/459#discussion_r117161326) [](ancestors = 117161326)